crates/accelerate/src/unitary_synthesis.rs

```diff
@@ -114,7 +116,7 @@ fn get_euler_basis_set(basis_list: IndexSet<&str>) -> EulerBasisSet {
 /// This will determine the available 1q synthesis basis for different decomposers.
 fn get_target_basis_set(target: &Target, qubit: PhysicalQubit) -> EulerBasisSet {
     let mut target_basis_set: EulerBasisSet = EulerBasisSet::new();
-    let target_basis_list = target.operation_names_for_qargs(Some(&smallvec![qubit]));
+    let target_basis_list = target.operation_names_for_qargs(TargetQargsRef::Concrete(&[qubit]));
```

Comment:
When I'm talking about the generics in the interfaces, what I mean is that it should be possible to specify _either `TargetQargsRef::Concrete(&[qubit])` here, or simply `&[qubit]`, and both should work.

Reply:
Added in 9602e7c